cdc_rsync/zstd_stream.h

```diff
@@ -39,6 +39,11 @@ class ZstdStream {
   // Flushes all remaining data and sends the compressed data to the socket.
   absl::Status Flush() ABSL_LOCKS_EXCLUDED(mutex_);
 
+  // Flush internal buffers if no new data is written for longer than this time.
+  // This makes sure that no data is stuck in the pipeline if no new input is
+  // available. Default is 500 ms.
+  void SetMinCompressPeriod(absl::Duration dur) { min_compress_period_ = dur; }
```

Comment:
I know, I know, it's all about naming with me, but I feel the identifiers you choose are not always aligned with the docstring you provide. In this example, why is it not called something with "flush", like `MaxFlushDelay` or `AutoFlushAfter`?

Reply:
I don't see why it's a problem to have two methods `Flush` and `AutoFlushAfter`, because they clearly tell what they do. Personally, I liked `Flush` better than `Finish` (more commonly called `Finalize`), but IIUC, it can be called multiple times, so it doesn't really finish the stream (in the finalize sense). No need to change it again, if you prefer to keep it.